### PR TITLE
French review of Introduction a MSupply

### PR DESCRIPTION
**Pull request by Cyrilbrites**
Premiere tentative. Je ne suis pas sur d'avoir tout compris a la procedure :) C'est une demande de revue des documents Open mSupply pour "Introduction"

### CODE REVIEW

File: content/docs/introduction/dashboard.fr.md [hunk 1]
```diff
@@ -14,29 +14,29 @@ toc = true
...
 - Expéditions
-  - **Aujourd'hui** Le nombre d'expéditions à préparer aujourd'hui. Il s'agit du nombre avec le statut `Allocated`
+  - **Aujourd'hui** Le nombre d'expéditions à préparer aujourd'hui. Il s'agit du nombre avec le statut `Alloué`
 - Stock
   - **Expiré(s)** Il s'agit du nombre de lignes de stock périmé pour ce dépôt.
   - **Stock expirant** Le nombre de ligne de stock à date de péremption proche (dans les 3 prochains mois)
   - **Total articles** Nombre total d'articles pour ce dépôt
-  - **Articles sans aucune quantité** Nombre total d'articles visibles n'ayant aucune quantité pour ce dépôt
-  - **Moins de 3 mois de stock** Basé sur la CMM pour cet article et pour ce dépôt
+  - **Articles sans aucune quantité** Nombre total d'articles visibles ayant une quantité nulle pour ce dépôt
```
Comment (Pauline-mSupply) [inline content/docs/introduction/dashboard.fr.md:38]: I feel like this may be a bit misleading. I would keep the original phrasing.

Comment (Pauline-mSupply) [inline content/docs/introduction/dashboard.fr.md:38]: I feel like this may be a bit misleading. I would keep the original phrasing.

**Review by Pauline-mSupply — CHANGES_REQUESTED** [review]

File: content/docs/introduction/dashboard.fr.md (hunk 1 above)
Comment (Cyrilbrites) [reply content/docs/introduction/dashboard.fr.md:38]: Ok, je suis tes recommandations

File: content/docs/introduction/introduction.fr.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ top = true
 
 ## Historique
 
-Open mSupply s'appuie sur plus de 20 années d'expériences enrichissantes de mSupply, au cours desquelles il est devenu le système d'information de gestion logistique (SIGL) le plus utilisé dans les pays à revenu faible et intermédiaire.
+Open mSupply s'appuie sur plus de 20 années d'expériencede mSupply, au cours desquelles il est devenu le système d'information de gestion logistique (SIGL) le plus utilisé dans les pays à revenu faible et intermédiaire.
```
Comment (Cyrilbrites) [inline content/docs/introduction/introduction.fr.md]:
```suggestion
Open mSupply s'appuie sur plus de 20 années d'expérience avec mSupply, au cours desquelles il est devenu le système d'information de gestion logistique (SIGL) le plus utilisé dans les pays à revenu faible et intermédiaire.
```

File: content/docs/introduction/introduction.fr.md [hunk 3]
```diff
@@ -36,18 +36,19 @@ Open mSupply fait bien plus que cela. Les fonctionnalités incluent:
   - Faire entrer les produits en stock après vérifications
 - Être alerté des niveaux de stock critiques depuis le tableau de bord
 
-## Qu'est-ce qui rend Open mSupply si spécial?
+## Les spécificités de Open mSupply
```
Comment (Cyrilbrites) [inline content/docs/introduction/introduction.fr.md:39]:
```suggestion
## Les spécificités d'Open mSupply
```

File: content/docs/introduction/introduction.fr.md [hunk 4]
```diff
@@ -36,18 +36,19 @@ Open mSupply fait bien plus que cela. Les fonctionnalités incluent:
...
-- **Evolutif** Nous avons conçu Open mSupply non seulement pour gérer des milliards de transactions par an, mais aussi pour avoir la possibilité de travailler sur une tablette Android! Vous pouvez implémenter mSupply dans un dépôt/entrepôt, sachant que si vous décidez par la suite de déployer des milliers de sites, mSupply sera toujours l'outil adéquat pour continuer.
+- **Hors ligne d'abord** mSupply est conçu pour fonctionner sans Internet. Même les connexions Internet les plus fiables peuvent se montrer instables voire s'interrompre. mSupply vous permet de travailler sans avoir à vous soucier de la qualité ou de la continuité d'Internet. Bvous avez besoin d'Internet pendant quelques minutes pour transmettre vos commandes ou recevoir des mises à jour.
```
Comment (Cyrilbrites) [inline content/docs/introduction/introduction.fr.md:43]:
```suggestion
- **Hors ligne d'abord** mSupply est conçu pour fonctionner sans Internet. Même les connexions Internet les plus fiables peuvent se montrer instables voire s'interrompre. mSupply vous permet de travailler sans avoir à vous soucier de la qualité ou de la continuité d'Internet. Vous avez besoin d'Internet pendant quelques minutes pour transmettre vos commandes ou recevoir des mises à jour.
```

File: content/docs/introduction/introduction.fr.md [hunk 5]
```diff
@@ -36,18 +36,19 @@ Open mSupply fait bien plus que cela. Les fonctionnalités incluent:
...
 
-Pour avoir un aperçu de l'étendue des fonctionnalités de mSupply, jetez un oeil par ici sur cet héritage [https://www.msupply.org.nz](https://www.msupply.org.nz) - C'est presque 1000 pages si vous imprimez tout (Alors n'essayez surtout pas ☺️)
+- **Evolutif** Nous avons conçu Open mSupply non seulement pour gérer des milliards de transactions par an, mais aussi pour avoir la possibilité de travailler sur une tablette Android! Vous pouvez mettre en place mSupply dans un dépôt/entrepôt et le déployer par la suite sur des milliers de sites. mSupply est l'outil adéquat pour toutes les échelles de projet.
+
+Pour avoir un aperçu de l'étendue des fonctionnalités de mSupply, jetez un oeil par ici sur cet héritage [https://www.msupply.org.nz](https://www.msupply.org.nz) - Il y a environ 1000 pages d'impression, n'essayez pas...
```
Comment (Cyrilbrites) [inline content/docs/introduction/introduction.fr.md:47]:
```suggestion
Pour avoir un aperçu de l'étendue des fonctionnalités de mSupply, jetez un oeil par ici sur notre version historique [https://www.msupply.org.nz](https://www.msupply.org.nz) - Il y a environ 1000 pages qui représentent 20 ans de travail , n'essayez pas d'imprimer...
```

File: content/docs/introduction/requirements.fr.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+[extra]
+toc = true
+top = true
++++
+
+## Configuration requise
+
+Open mSupply fonctionne sur un certain nombre de plateformes et a des besoins matériels très modestes pour une installation de base.
+Le client peut être un navigateur web, une tablette Android ou un PC sous Windows ou macOS.
```
[the remark this answers is not in the document]
Comment (Cyrilbrites) [reply content/docs/introduction/requirements.fr.md:19]: A priori oui

File: content/docs/introduction/requirements.fr.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+- Une imprimante connectée n'est pas absolument nécessaire dans toutes les situations, mais elle l'est dans la plupart des cas. Une imprimante à jet d'encre ou laser est préférable.
+- Achetez du matériel pour lequel un bon service après-vente est disponible.
+- Achetez des marques réputées pour leur qualité, même si cela signifie acheter une machine plus lente/ancienne.
+
+### Configuration requise pour Windows, Mac et Linux
+
+La configuration matérielle requise varie considérablement en fonction de l'utilisation prévue d'Open mSupply. Voici un guide informatif (en Anglais):
+
+| Machine                                    | Technical Minimum                                                                                                                                              | Recommended                                                                                                                                                                                                                              |
```
[the remark this answers is not in the document]
Comment (Cyrilbrites) [reply content/docs/introduction/requirements.fr.md:42]: Oui, je suis d'accord mais je pense que c'est comprehensible meme pour un public non anglophone

**Review by Cyrilbrites — COMMENTED** [review]
Let me know when you can if you are happy with the changes suggested. I am going to keep sending suggestions for other sections + screenshots with the French version where I can.

**Pull request closed without merging**